src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RADIUS = 10;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int FIGURE_COUNT = 5;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    private Figure getSquare() {
+        int side = random.nextInt(RADIUS);
+        return new Square(colorSupplier.getRandomColor(), side);
+    }
+
+    private Figure getRectangle() {
+        int length = random.nextInt(RADIUS);
+        int width = random.nextInt(RADIUS);
```

Comment:
a rectangle doesn't have a radius, so this code looks strange. rename constant to something like MAX_SIZE